build.sbt

```diff
@@ -17,12 +17,12 @@ lazy val root = (project in file(".")).
     scriptClasspathOrdering := Seq.empty,
     BundleKeys.system := "postgres94",
     BundleKeys.bundleType := Docker,
-    BundleKeys.nrOfCpus := 4.0,
+    BundleKeys.nrOfCpus := 2.0,
     BundleKeys.memory := 2.GB,
     BundleKeys.diskSpace := 10.GB,
     BundleKeys.roles := Set("postgres94"),
     BundleKeys.endpoints := Map(
-      "postgres" -> Endpoint("tcp", 5432, services = Set(uri("tcp://:5432")))
+      "postgres" -> Endpoint("tcp", 5432, "postgres", RequestAcl(Tcp(5432)))
```

Comment:
This will mean no more compatibility with 1.1. I think we should retain services here.